models/migrations/migrations.go

```diff
@@ -491,8 +491,12 @@ var migrations = []Migration{
 	NewMigration("Add ArchivedUnix Column", v1_20.AddArchivedUnixToRepository),
 	// v256 -> v257
 	NewMigration("Add is_internal column to package", v1_20.AddIsInternalColumnToPackage),
+	// v258 -> 259
+	NewMigration("Add PinOrder Column", v1_20.AddPinOrderToIssue),
 	// v257 -> v258
 	NewMigration("Add Actions Artifact table", v1_20.CreateActionArtifactTable),
+	// v258 -> 259
+	NewMigration("Add PinOrder Column", v1_20.AddPinOrderToIssue),
```

Comment:
Oops, the annoying Gitea's migration framework ..... 😭 

Otherwise, overall LGTM.

Reply:
What's wrong with this line?

Reply:
Merge problem.

![image](https://github.com/go-gitea/gitea/assets/2114189/ba50b14e-bbef-4ccb-9b37-0c5b6a19264d)